subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -1248,6 +1248,16 @@ config BT_CTLR_USER_CPR_ANCHOR_POINT_MOVE
 
 endmenu
 
+# Workaround to be able to have default for "choice" in hidden "menu" above
+choice BT_CTLR_OPTIMIZE
+	prompt "Optimization options"
+	depends on !LTO
+
+config BT_CTLR_OPTIMIZE_FOR_SPEED
+	bool "Optimize for Speed"
+
+endchoice
```

Comment:
Can you elaborate? How is this a workaround? Why is this needed?

Reply:
Refer to: https://github.com/zephyrproject-rtos/zephyr/issues/39756?imz_s=gpfpb5qgq9s43md0esvadt4d47#issuecomment-953561685

> only visible config items inside the choice is selectable

Reply:
:facepalm: